File: 프로그래머스/unrated/120907. OX퀴즈/OX퀴즈.java
import java.util.*;

class Solution {
    public String[] solution(String[] quiz) {
        String[] answer = new String[quiz.length];
        int res = 0;
        for(int i = 0; i<answer.length; i++){
            String[] strArr = quiz[i].split(" ");
            res = Integer.parseInt(strArr[0]) + Integer.parseInt(strArr[2]) * (strArr[1].equals("+") ? 1 : -1);
            answer[i] = res == Integer.parseInt(strArr[4]) ? "O" : "X";
        }
        return answer;
    }
}

File: 프로그래머스/Lv.0/120907. OX퀴즈/OX퀴즈.js
function solution(quiz) {
    var answer = [];
    for(let i = 0; i<quiz.length; i++){
        let str = eval(quiz[i].split(" = ")[0]) == quiz[i].split(" = ")[1] ? "O" : "X";
        answer.push(str);
    }
    return answer;
}